client/src/components/graphs/OrganizationalChart.tsx

```diff
@@ -95,90 +103,44 @@ const GQL_GET_CHART_DATA = gql`
     }
   }
 `
-const transitionDuration = 200
-
-const roles = Object.keys(PositionRole)
-const ranks = Settings.fields.person.ranks.map(rank => rank.value)
-
-const sortPositions = (positions, truncateLimit) => {
-  const allResults = positions?.sort(
-    (p1, p2) =>
-      // highest position role first
-      roles.indexOf(p2.role) - roles.indexOf(p1.role) ||
-      // when these are equal, highest ranking person first
-      ranks.indexOf(p2.person?.rank) - ranks.indexOf(p1.person?.rank) ||
-      // when these are also equal, sort alphabetically on person name
-      p1.person?.name?.localeCompare(p2.person?.name) ||
-      // else sort by position name
-      p1.name?.localeCompare(p2.name) ||
-      // last resort: sort by position uuid
-      p1.uuid.localeCompare(p2.uuid)
-  )
 
-  return allResults.slice(0, truncateLimit)
+const BACKGROUND_COLOR = "#fafafa"
+const ORG_AVATAR_WIDTH = 60
+const TEXT_WIDTH = 200
+const TEXT_GAP = 10
+const NODE_WIDTH = ORG_AVATAR_WIDTH / 2 + TEXT_GAP + TEXT_WIDTH
+const NODE_HEIGHT = 60
+const VERTICAL_SPACING = 20
+const HORIZONTAL_SPACING = 10
+const LEVEL_INDENT = 60
+const PERSON_AVATAR_HEIGHT = 42
+type PeopleFilterOption =
+  | "none"
+  | "leaders"
+  | "leaders_deputies"
+  | "highest_rank"
+  | "highest_2_ranks"
```

Comment:
These two should use the old ordering (so leaders then deputies then members ordered by rank), and then limit the number of positions displayed to 1 or 2.